qa/tomcat-runtime/pom.xml

```diff
@@ -40,6 +40,52 @@
       <scope>runtime</scope>
     </dependency>
 
+    <dependency>
```

Comment:
Could you use the Testcontainers BOM
```
  <dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>org.testcontainers</groupId>
        <artifactId>testcontainers-bom</artifactId>
        <version>${version.testcontainers}</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
    </dependencies>
  </dependencyManagement>
```
Then leave our `<version>` tags here.

Also update the version in parent/pom.xml

Reply:
Good catch!